gui/default/assets/css/overrides.css

```diff
@@ -338,6 +338,12 @@ ul.three-columns li, ul.two-columns li {
     z-index: 980;
 }
 
+
+.login-form-buttons {
+  padding: 0;
+  text-align: right;
+}
+
```

Comment:
What's the reason for this override, don't the Bootstrap styles suffice?

And I think you added an unintentional blank line there.

Reply:
Good catch, thanks!

Reply:
Fixed in 417170e2b1b8252d32d5c55c829fcfd9b4dc00ee.